scripts/sanitycheck

```diff
@@ -2096,413 +2505,261 @@ class TestSuite:
-                        defconfig_path = os.path.join(o, "zephyr", ".config")
-                        dt_list[tc, plat, tc.name.split("/")[-1]] = dt_config_path
-                        cmake_list[tc, plat, tc.name.split("/")[-1]] = cmake_cache_path
-                        defconfig_list[tc, plat, tc.name.split("/")[-1]] = defconfig_path
-                        goal = "_".join([plat.name, "_".join(tc.name.split("/")), "config-sanitycheck"])
-                        tc_file_list[goal] = (o, [dt_config_path, cmake_cache_path, defconfig_path])
-                        mg.add_build_goal(goal, os.path.join(ZEPHYR_BASE, tc.test_path),
-                                o, args, "config-sanitycheck.log", make_args="config-sanitycheck")
-
-        info("Building testcase defconfigs...")
-        kept_files = tc_file_list if options.save_tests else None
-        results = mg.execute(defconfig_cb, keep_files=kept_files)
-
-        info("Filtering test cases...")
-        for name, goal in results.items():
-            try:
-                if goal.failed:
-                    raise SanityRuntimeError("Couldn't build some defconfigs")
-            except Exception as e:
-                error(str(e))
-                sys.exit(2)
-
-
-        for k, out_config in defconfig_list.items():
-            test, plat, name = k
-            defconfig = {}
-            with open(out_config, "r") as fp:
-                for line in fp.readlines():
-                    m = TestSuite.config_re.match(line)
-                    if not m:
-                        if line.strip() and not line.startswith("#"):
-                            sys.stderr.write("Unrecognized line %s\n" % line)
-                        continue
-                    defconfig[m.group(1)] = m.group(2).strip()
-            test.defconfig[plat] = defconfig
-
-        for k, cache_file in cmake_list.items():
-            if not os.path.exists(out_config):
-                continue
-
-            test, plat, name = k
-            cmake_conf = {}
-            try:
-                cache = CMakeCache.from_file(cache_file)
-            except FileNotFoundError:
-                cache = {}
-
-            for k in iter(cache):
-                cmake_conf[k.name] = k.value
-
-            test.cmake_cache[plat] = cmake_conf
-
-        for k, out_config in dt_list.items():
-            if not os.path.exists(out_config):
-                continue
-
-            test, plat, name = k
-            dt_conf = {}
-            with open(out_config, "r") as fp:
-                for line in fp.readlines():
-                    m = TestSuite.dt_re.match(line)
-                    if not m:
-                        if line.strip() and not line.startswith("#"):
-                            sys.stderr.write("Unrecognized line %s\n" % line)
-                        continue
-                    dt_conf[m.group(1)] = m.group(2).strip()
-            test.dt_config[plat] = dt_conf
+        info("Building initial testcase list")
```

Comment:
Add ellipsis "..." at the end since this indicates that this is what the system is currently doing